setup/README.md

```diff
@@ -1,419 +1,21 @@
-# A Connaisseur Setup Guide
+# Setup
 
-## Intro
+Setting up Connaisseur for the first time can be a bit of a hassle, especially if you are not familiar with Docker Content Trust (DCT). Hence, we've created multiple guides for various environments to help you on the way.
```

Comment:
```suggestion
Setting up Connaisseur for the first time can be a bit of a hassle, especially if you are not familiar with Docker Content Trust (DCT). Furthermore, Connaisseur requires/links several other services (Kubernetes cluster, container registry, notary server) for which many common providers exist. Hence, we've created multiple guides for various environments to help you on the way.

The guide below offers a simple default configuration for setting up Connaisseur using public infrastructure and is aimed at quick testing. It uses [Docker Hub](https://hub.docker.com/) as both container registry and notary, but is expected to work for other managed solutions as well. It has been tested for the following Kubernetes services:
- [x] [minikube](https://github.com/kubernetes/minikube)
- [x] [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/)
- [x] [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/)
- [ ] [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/)
- [ ] [SysEleven MetaKube](https://docs.syseleven.de/metakube/de)

Two further specialized guides exist:
1. **Local setup ([Minikube](https://github.com/kubernetes/minikube) & [Harbor](https://github.com/goharbor/harbor))**: Connaisseur can be run completely locally with a minikube instance paired with a local Harbor installation that combines registry and notary servers. This setup is a bit more cumbersome since it does not use existing public infrastructure such as PKI. However, it is entirely based on open-source solutions, does not require continuous access to the internet and you control every part of the installation. This makes it specifically suited e.g. for security researchers and users interested in the inner workings of DCT and Connaisseur. Head over to [the respective README](local/README.md) to try it out.
2. TODO: **Azure [AKS](https://docs.microsoft.com/en-us/azure/aks/) with [Azure Container Registry (ACR)](https://docs.microsoft.com/en-us/azure/container-registry/)**: Microsoft Azure is currently the only one of the big cloud providers with a managed Kubernetes and container registry that supports DCT.
```
Explains why we offer separate guides, which ones were tested and other approaches without making this too deep a guide. I would then drop the other headings below and put the 'EKS Guide' directly in here below. This makes the setup guide flatter.